extension/webpack.dev.js

```diff
@@ -14,6 +15,14 @@ const devConfig = {
     new webpack.DefinePlugin({
       DEVELOPMENT: true,
     }),
+    new ModuleReplaceWebpackPlugin({
+      modules: [
+        {
+          test: /webextension-polyfill-ts/,
+          replace: "./src/shims/webextension-polyfill.ts",
+        },
```

Comment:
oh boy - so `webextension-polyfill` tries to be helpful and will throw a breaking error if you try to run it in a browser (instead of an extension). Unfortunately, for our dev flow, we need to open the extension in a browser. We don't _use_ the webextension in dev mode, but just importing the lib crashes the app. Very annoying. Very lame solution